storage/statsd/statsd.go

```diff
@@ -31,6 +31,12 @@ type statsdStorage struct {
 
 const (
 	colCpuCumulativeUsage string = "cpu_cumulative_usage"
+	// CPU system
+	colCpuUsageSystem string = "cpu_usage_system"
+	// CPU user
+	colCpuUsageUser string = "cpu_usage_user"
+	// CPU average load
+	colLoadAverage string = "load_average"
```

Comment:
I don't have a problem with that. I can get it updated when I get home from work.

Thanks